app/messaging/fetch-application.js

```diff
@@ -8,6 +8,12 @@ const fetchApplication = async (message) => {
     const msgBody = message.body
     console.log('received application fetch request', util.inspect(msgBody, false, null, true))
     const application = await get(msgBody.applicationReference)
+
+    // if application doesn't exists or already submitted return null.
+    if (!application || application?.vetVisit?.dataValues) {
+      return sendMessage(null, fetchApplicationResponseMsgType, applicationResponseQueue, { sessionId: msgBody.sessionId })
```

Comment:
Same as `process-vet-visit`, can a state be sent back in the message so the frontend can show a more informed message please.